revdb/src/sqlite_connection.cpp

```diff
@@ -182,6 +182,10 @@ Bool DBConnection_SQLITE::sqlExecute(char *query, DBString *args, int numargs, u
 		}
 
 		int rv = basicExec(newquery, &affectedrows);
+        
+        if (numargs > 0)
+            free(newquery);
```

Comment:
Look at the definition of `BindVariables()` in `revdb/src/revdb.cpp`. The eventual return value (which is assigned to `newquery`) in this code) comes from `t_values`.  `t_values` is always allocated with `new[]` (on revdb.cpp:585 or revdb.cpp:635).

So, if `newquery` is assigned using `BindVariables()`, then it should be freed with `delete[]`.